### PR TITLE
Pricing Section updated in Popular APIs panel

### PR DESCRIPTION
**Pull request by Vinyl-Davyl**
Awaiting Single API section

### CODE REVIEW

**Review by pablo-clueless — COMMENTED** [review]
- Create an images folders in the assets folder for images
- Why `.webp` for images?? Use `.png`, `.jpg` or `.svg`

You can use this site to convert
https://convertio.co

**Comment by Vinyl-Davyl** [comment]
> * Create an images folders in the assets folder for images
> * Why `.webp` for images?? Use `.png`, `.jpg` or `.svg`
> 
> You can use this site to convert https://convertio.co

The main reason I used .webp was because of the size it store images (mostly 1.23kb - 5kb or less), while .png , .jpg and the likes use to be higher

>>
>>But everything is fine now, Its all good now.

File: src/components/Pricing.tsx [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+import apicalls from "../assets/images/apicalls.jpg";
+import countries from "../assets/images//countries.jpg";
+import sponsors from "../assets/images/sponsors.jpg";
+import charts from "../assets/images/charts.jpg";
+
+
+const Pricing:React.FC = () => {
+    const classes = useStyles()
+
+    const [open, setOpen] = React.useState(true);
```
Comment (pablo-clueless) [inline src/components/Pricing.tsx]: Add a type boolean to the useState, like so:

`React.useState<boolean>(true)`

File: src/components/Pricing.tsx [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+
+    const handleClick = () => {
```
Comment (pablo-clueless) [inline src/components/Pricing.tsx]: At a type, Synthetic event here, like so:

```
const handleClick = (e: SynthenticEvent) => {

**Review by taiwoAkindele — APPROVED** [review]

**Review by deviator53 — APPROVED** [review]

**Review by pablo-clueless — APPROVED** [review]

**Pull request merged**